ledger/controllers/surveyor.js

```diff
@@ -36,10 +37,45 @@ const server = async (request, reply, runtime) => {
     surveyor.payload = entry.payload
     surveyor.parentId = entry.parentId
   }
+  return surveyor && addChoices(runtime, surveyor)
+}
 
+async function addChoices (runtime, surveyor) {
+  const payload = surveyor.payload || {}
+  surveyor.payload = payload
+  const adFree = payload.adFree || {}
+  payload.adFree = adFree
+  adFree.choices = await getAdjustedChoices(runtime, 'USD', ['BAT'])
   return surveyor
 }
 
+async function getAdjustedChoices (runtime, base, currencies) {
+  const rates = await runtime.currency.rates(base, currencies)
+  return underscore.mapObject(rates, choicesPrices)
+}
+
+function choicesPrices (ratio) {
+  const basePrice = (new BigNumber(1)).dividedBy(ratio)
```

Comment:
If we use BAT as the base in getAdjustedChoices can we remove this line here?

Reply:
I'm assuming the rates that are returned are in terms of 1 unit of the base.  In this case since USD is the base, I would think that the rate would be how many BAT is 1 USD, right now about 6.4 BAT. 

So the basePrice as defined would be 1/6.4 or .156.  But that kind of confuses me since the price of 1 USD is not .156 BAT.  Where am I going wrong?

Reply:
right so we're just trying to detect the closest approximated values and show the browser our estimate for how much they should be asking for (in usd) for it to be reasonable, or rather, the closest approximation of prices. 

I find that a good way to think about the ratios is as follows: how many Y can i get with X. With 1 USD, I can get 6.4 BAT. With 1 BAT I can get 0.156 USD. 

I decided to keep it simple and directly use the values that mandar had provided in his spreadsheet. This was so that if there were updates. I would not have to go through the step of transforming the data before putting it into the repo. I cannot currently find the spreadsheet, but I thought it would be better to just use the numbers from his point of view so that he could continue to think about it the same way.

Reply:
Thanks @maikelmclauflin, that makes sense.  I guess my question is if USD is the base, then why is the basePrice .156 BAT.  Isn't the price of 1 USD 6.4 BAT?

Reply:
ah, yeah, after re-reading your quest, i guess we could. never thought of that. can modify